core/unit_test/tstNeighborList.hpp

```diff
@@ -84,19 +84,19 @@ convert_crs_graph_to_verlet_list(
             auto const last = crs_graph.row_ptr( i + 1 );
             counts( shift + i ) = last - first;
         } );
-    int const max_entries_per_row = ArborX::max( counts );
+    using ExecutionSpace = typename DeviceType::execution_space;
+    int const max_entries_per_row = ArborX::max( ExecutionSpace{}, counts );
```

Comment:
is this your own portable implementation of `std::max`?

Reply:
No,  it performs a parallel reduce and returns the maximum element in a view.